example/src/pages/LinePage.tsx

```diff
@@ -35,21 +35,40 @@ const Line: React.FC<{}> = () => {
   });
 
   const fetchLine = useCallback(async () => {
-    const state = [{
-      name: 'Line 1',
+    const state: IPointSet[] = [{
+      name: 'UP',
       points: [
           {
             x: 100,
-            y: 100
+            y: 200
           },
           {
-            x: 200,
+            x: 300,
             y: 200
           }
         ],
         readOnly: false,
-        styling: "primary"
-      }
+        styling: 'primary',
+        centerIcon: ArrowIcon,
+        centerIconPosition: 'up'
+      },
+      {
+        name: 'DOWN',
+        points: [
+            {
+              x: 200,
+              y: 400
+            },
+            {
+              x: 400,
+              y: 400
+            }
```

Comment:
Related to other comments on implementation of up / down. Once that has been resolved you may want to flip these points so the arrow appears on the opposite side of the line.